alerter/src/main/java/com/tapadoo/alerter/Alert.kt

```diff
@@ -0,0 +1,576 @@
+package com.tapadoo.alerter
```

Comment:
add an `@file:JvmName("")` to any public files so Java projects can still call them.